src/zkevm_specs/evm/execution/callop.py

```diff
@@ -44,57 +43,37 @@ def callop(instruction: Instruction):
     # Verify depth is less than 1024
     instruction.range_lookup(depth, 1024)
 
-    # Lookup values from stack
-    gas_rlc = instruction.stack_pop()
-    code_address_rlc = instruction.stack_pop()
-    # The third stack pop `value` is not present for both DELEGATECALL and
-    # STATICCALL opcodes.
-    value = instruction.stack_pop() if is_call + is_callcode == 1 else RLC(0)
-    cd_offset_rlc = instruction.stack_pop()
-    cd_length_rlc = instruction.stack_pop()
-    rd_offset_rlc = instruction.stack_pop()
-    rd_length_rlc = instruction.stack_pop()
-    is_success = instruction.stack_push()
-
-    # Verify is_success is a bool
-    instruction.constrain_bool(is_success)
-
-    # Recomposition of random linear combination to integer
-    code_address = instruction.rlc_to_fq(code_address_rlc, N_BYTES_ACCOUNT_ADDRESS)
-
+    call_data = common_call_stack_pop(instruction, FQ(0), is_call + is_callcode)
```

Comment:
The name `call_data` collides with the EVM `calldata`.
What do you think we name it `call` directly here? It would make variable names like `call.callee_address` and `call.value` look natural.

Reply:
fixed in [ad22495](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/348/commits/ad224959a3e90df0f79898b52888fab521c80f8b)